examples/dt/uni02beta/control-plane/nncp/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../../../lib/nncp
+  - ../../../../../lib/networking/metallb
+  - ../../../../../lib/networking/netconfig
+  - ../../../../../lib/networking/nad
```

Comment:
I've noticed in other DTs that people are placing these with the control plane `kustomization.yaml` (and keeping `nncp` separate here).  Maybe we should follow the same pattern consistently?

Reply:
Moved to control-plane kustomization as suggested, thanks!